signalfx-org-metrics/metrics.yaml

```diff
@@ -1693,8 +1704,18 @@ sf.org.subscription.highResolutionMetrics:
 sf.org.subscription.hosts:
   brief: Number of hosts.
   description: |
-    Number of hosts.
+    Number of hosts included in the subscription.
     
       * Data resolution: 10 seconds
   metric_type: gauge
   title: sf.org.subscription.hosts
+ 
+sf.org.subscription.function:
```

Comment:
Do we already define "serverless function"?

Reply:
Good question. "Serverless function" is detailed in the docs site, but seems to refer only to AWS Lambda functions.